aspnet/mvc/views/working-with-forms.rst

```diff
@@ -0,0 +1,486 @@
+Working with Forms
+============================
+
+By `Rick Anderson`_, `Dave Paquette <https://twitter.com/Dave_Paquette>`_ and `Jerrie Pelser <https://twitter.com/jerriepelser>`__
+
+This document demonstrates working with Forms and the HTML elements commonly used on a Form. The HTML `Form <https://www.w3.org/TR/html401/interact/forms.html>`__ element provides the primary mechanism web apps use to postback data to the server. Most of this document describes :doc:`Tag Helpers <tag-helpers/intro>` and how they can help you productively create robust HTML forms. We recommend you read :doc:`tag-helpers/intro` before you read this document.
+
+In many cases, :doc:`HTML Helpers </mvc/views/html-helpers>` provide an alternative approach to a specific Tag Helper, but it's important to recognize that Tag Helpers do not replace HTML Helpers and there is not a Tag Helper for each HTML Helper. When an HTML Helper alternative exists, it is mentioned.
+
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+The Form Tag Helper
+---------------------
+  
+The `Form <https://www.w3.org/TR/html401/interact/forms.html>`__ Tag Helper:
+
+- Generates the HTML `<FORM> <https://www.w3.org/TR/html401/interact/forms.html>`__ ``action`` attribute value for a MVC controller action or named route
+- Generates a hidden `Request Verification Token <http://www.asp.net/mvc/overview/security/xsrfcsrf-prevention-in-aspnet-mvc-and-web-pages>`__ to prevent cross-site request forgery
+- Supports the Tag Helper attribute ``asp-route-returnurl`` which provides a return URL
+- HTML Helper alternative: ``Html.BeginForm``
```

Comment:
- Suggest "Has an HTML Helper alternative ..."
- Be consistent. Generally like these bullets and miss them where you've left them out.